projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_dag.py

```diff
@@ -12,12 +12,56 @@
 from typing import List
 
 from taurus_datajob_api import ApiException
+from vdk.internal.core import errors
 from vdk.plugin.meta_jobs.cached_data_job_executor import TrackingDataJobExecutor
 from vdk.plugin.meta_jobs.meta import TrackableJob
 from vdk.plugin.meta_jobs.remote_data_job_executor import RemoteDataJobExecutor
 from vdk.plugin.meta_jobs.time_based_queue import TimeBasedQueue
 
 log = logging.getLogger(__name__)
+max_starting_jobs = 15
+
+
+def validate_job_limit(jobs: List[Dict]):
+    topo_sorter = TopologicalSorter()
+    for job in jobs:
+        topo_sorter.add(job["job_name"], *job["depends_on"])
+
+    sorted_jobs = []
+    for job_name in topo_sorter.static_order():
+        job = next((j for j in jobs if j["job_name"] == job_name), None)
```

Comment:
Is there a possibility that "depends_on" job name is not in `jobs` list (param) keys? 
For example:
```
A -(depends on)-> B
C - ...
```
How so (asking about the logic here), and should we at least log a message